signac/syncutil.py

```diff
@@ -15,14 +20,30 @@
 
 
 def log_more(msg, *args, **kwargs):
+    """Log using LEVEL_MORE."""
     logger.log(LEVEL_MORE, msg, *args, **kwargs)
 
 
 logger.more = log_more  # type: ignore
 
 
 def copytree(src, dst, copy_function=shutil.copy2, symlinks=False):
-    "Implementation adapted from https://docs.python.org/3/library/shutil.html#copytree-example'."
+    """Recursively copy a directory tree from src to dst, using a custom copy function.
+
+    Implementation adapted from https://docs.python.org/3/library/shutil.html#copytree-example.
+
+    Parameters
+    ----------
+    src :
+        Source directory tree.
+    dst :
```

Comment:
if I'm reading the code correctly:
```suggestion
    dst : str
```